docs/modules/ROOT/pages/user-guide/native-mode.adoc

```diff
@@ -25,6 +25,22 @@ quarkus.native.add-all-charsets = true
 See also https://quarkus.io/guides/all-config#quarkus-core_quarkus.native.add-all-charsets[quarkus.native.add-all-charsets]
 in Quarkus documentation.
 
+[[locale]]
+== Locale
+
+Before GraalVM 21.1 there was no way to include more than the single locale (the one set as a defult in the building JVM) in the native image. 
```

Comment:
What about turning it the positive way, like:
Before GraalVM 21.1, only the building JVM default locale was included in the native image.

Reply:
+1 less words is better :)